src/render/shaders/Textures.hpp

```diff
@@ -309,17 +197,17 @@ uniform vec2 halfpixel;
 void main() {
 	vec2 uv = v_texcoord / 2.0;
 
-    vec4 sum = texture2D(tex, uv + vec2(-halfpixel.x * 2.0, 0.0) * radius);
+        vec4 sum = texture2D(tex, uv + vec2(-halfpixel.x * 2.0, 0.0) * radius);
```

Comment:
this wasnt intended I think

Reply:
Shouldn't it be 8 spaces?

Reply:
why??? it's the first indent level, so 4.

Reply:
But look inside that shader, its everywhere 8 inside the string, even the vec2 uv... above that.
I was trying to keep the style.

Reply:
weird, it looks really off on my end. Defo wasnt intended.

```
void main() {
    // 4 spaces
        // 8 spaces
}
```

Reply:
Is this ok though?

Reply:
na make it 4

Reply:
the whole file?

Reply:
wherever you changed it, the rest seems to be 4 to me

Reply:
One second!

Reply:
just for the record, this file wasn't 4 spaces before.
Next time i find a file like that i let you fix it before i touch it !

Reply:
Your resolve this conversation.

Reply:
no clue then. In my vscode it was 4 spaces. weird shit.

Reply:
Man i kind of get used to the tone in this repo!

Reply:
good